tutorials/BrowserView.EventEmitter.md

```diff
@@ -72,54 +72,91 @@ view.removeAllListeners("closed");
 
 ### Supported BrowserView event types
 
-* created
 * attached
-* shown
-* hidden
+* crashed
+* created
+* did-change-theme-color
 * destroyed
+* hidden
+* page-favicon-updated
+* page-title-updated
+* shown
 
 ### BrowserView Events
 
-#### created
-Generated when a BrowserView is created.
+
+#### attached
+Generated when a BrowserView attaches to a window. This event will fire during creation of a BrowserView. In that case, `previousTarget` identity will be the same as `target` identity.
 ```js
 //This response has the following shape:
 {
     name: "BrowserViewName" // the name of the BrowserView
+    previousTarget: {uuid: 'previousWindowUuid', name: 'previousWindowName'}, // the window this BrowserView was previously attached to
     target: {uuid: 'windowUuid', name: 'windowName'}, // the window this BrowserView will attach to
     topic: "view",
-    type: "created",
+    type: "attached",
     uuid: "454C7F31-A915-4EA2-83F2-CFA655453C52" // the uuid of the BrowserView
 }
 ```
 
-#### attached
-Generated when a BrowserView attaches to a window. This event will fire during creation of a BrowserView. In that case, `previousTarget` identity will be the same as `target` identity.
+#### crashed
+Generated when a view has crashed.
+```js
+//This response has the following shape:
+{
+    name: "windowOne", //the name of the window.
+    reason: "killed", //possible values:
```

Comment:
name is  browserView name, not window name?

Reply:
Updated!